runtime/parser2/parser.go

```diff
@@ -244,8 +255,22 @@ func (p *parser) replayBuffered() {
 	p.next()
 }
 
-func (p *parser) skipSpaceAndComments(skipNewlines bool) (containsNewline bool) {
```

Comment:
>How do you propose to refactor the function?

Instead of removing it, we could refactor this function with `parseTrivia`:

```golang
func (p *parser) skipSpaceAndComments(skipNewlines bool) (containsNewline bool) {
  containsNewline, _ := p.parseTrivia(triviaOptions{
    skipNewlines: skipNewlines,
  })
  return containsNewline
}
```

So that we don't need to change `skipSpaceAndComments` everywhere.

Reply:
Great idea 👍

Reply:
Added back in 3360414307707b32fb31232dba804023af9ec56f